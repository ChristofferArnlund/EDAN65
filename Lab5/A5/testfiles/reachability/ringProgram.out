reach(a) = {a, b, c, d, e, f}
reach(b) = {a, b, c, d, e, f}
reach(c) = {a, b, c, d, e, f}
reach(d) = {a, b, c, d, e, f}
reach(e) = {a, b, c, d, e, f}
reach(f) = {a, b, c, d, e, f}
