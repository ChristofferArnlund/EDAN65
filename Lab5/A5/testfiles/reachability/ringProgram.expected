reach(a) = {b, c, e, a, f, d}
reach(b) = {c, b, e, a, f, d}
reach(c) = {b, c, e, a, f, d}
reach(d) = {b, c, e, a, f, d}
reach(e) = {b, c, a, e, f, d}
reach(f) = {b, c, a, e, f, d}
